dockstore-webservice/src/main/java/io/dockstore/webservice/helpers/GitHubHelper.java

```diff
@@ -42,10 +47,50 @@
 public final class GitHubHelper {
 
     private static final Logger LOG = LoggerFactory.getLogger(GitHubHelper.class);
+    public static final String BRANCHNAME_FOR_BOT = "feature/add_dockstore_yml";
+
+    public static final String DOCKSTORE_BOT_PR_TEXT = """
+           Dockstore's bot has guessed at a .dockstore.yml for your repository.
```

Comment:
Might as well pass the branch name and PR title/test as arguments to the method, that way we can easily tweak the message, and could use the method to create other types of PRs, someday...